Amethyst/Managers/WindowManager.swift

```diff
@@ -187,6 +187,20 @@ private class ObserveApplicationNotifications {
                 windowManager.addWindow(window)
             }
 
+            application.observeNotification(kAXApplicationHiddenNotification as CFString!, with: application) { accessibilityElement in
```

Comment:
Seems like this should be at the application level rather than the window level. If each window is observing then an application visibility change would trigger one event for each window.

Reply:
Seeking comments @ianyh ^

Reply:
Bump

Reply:
I think i was actually mistaken in my assessment. I think your code is correct.